datalad/resources/website/assets/js/main.js

```diff
@@ -205,8 +205,9 @@ function directory(jQuery, md5) {
     // add click handlers to each row(cell) once table initialised
     initComplete: function() {
       var api = this.api();
-      // all tables should have ../ parent path
-      if (!parent)
+      // all tables should have ../ parent path except webinterface root
+      var current_loc = absolute_url(getParameterByName('dir')).replace(/\/?$/, '') + '/';
```

Comment:
I think it would be valuable to create a treat repo with some directory which would use obscure filename as we use in tests on python side and then see how above substitution plays out (is there URL encoding happening anywhere etc)

But you did test this locally at least didn't you?